go/ndt7-client/sink/reader.go

```diff
@@ -0,0 +1,48 @@
+package sink
+
+import (
+	"strings"
+	"time"
+
+	"github.com/apex/log"
+	"github.com/gorilla/websocket"
+)
+
+// logmeasurement logs a measurement received from the server. We trim the
+// trailing newlines such that the logs are more compact.
+func logmeasurement(data []byte) {
+	log.Infof("%s", strings.TrimRight(string(data), "\n"))
+}
+
+// Reader reads messages from the websocket connection in a background
+// goroutine. The length of messages will be posted on the returned
```

Comment:
"The length of each message will be"

Reply:
The code seems to be changed, I'll check and see whether the new code also has this issue.